app/javascript/src/components/editor/EditorActions.svelte

```diff
@@ -6,8 +6,23 @@
   import Settings from "./Settings.svelte"
   import Shortcuts from "./Shortcuts.svelte"
   import Save from "./Save.svelte"
+  import ThreeDotMenu from "../icon/ThreeDotMenu.svelte"
+
+  let mobileDropdown
+  let showMobileDropdown = false
+
+  $: if (!$isMobile) showMobileDropdown = false
```

Comment:
Is the idea of this to set it to false if you change from mobile to desktop while the menu is open?

Reply:
Indeed. Just a small thing to close the dropdown when it won't be shown anyway, so if the user changes from desktop to mobile again the dropdown isn't still open.